ui/aboutPC.html

```diff
@@ -1,23 +1,5982 @@
-<!-- 
-Version : Phython/Django 2.7.6, PostgreSQL 9.3.4
-Author : Vaibhavi Desai
-Github username : desaivaibhavi
-email : ranihaileydesai@gmail.com
--->
-
-<!-- page for about us -->
-<html>
-<head>
-	<title> About PC </title>
-</head>
-<body>
-    {% include "header.html" %}
-    <br><br><br><br><br><br><center>
-    <p>Here the information about PC would be shown</p></center>
-    
-    <br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br>
-    {% include "footer.html" %}
-    
-</body>
-</html>
-
+<!-- 
+Version : Phython/Django 2.7.6, PostgreSQL 9.3.4
+Author : Vaibhavi Desai
+Github username : desaivaibhavi
+email : ranihaileydesai@gmail.com
+-->
+
+<!-- page for about us -->
+<html>
+    <head>
+        <title> About PC </title>
+
+        <style>

```

Comment:
Can you paste the code you wrote for including the external CSS, here? I can take a look.